<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>stage1</class>
 <widget class="QDialog" name="stage1">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>450</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QColumnView" name="columnView">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>150</width>
     <height>450</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(173, 127, 168);</string>
   </property>
  </widget>
  <widget class="QPushButton" name="restart_butoon">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>280</y>
     <width>150</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(114, 159, 207);</string>
   </property>
   <property name="text">
    <string>RESTART</string>
   </property>
  </widget>
  <widget class="QPushButton" name="menu_button">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>350</y>
     <width>150</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(114, 159, 207);</string>
   </property>
   <property name="text">
    <string>MENU</string>
   </property>
  </widget>
  <widget class="QLabel" name="score_label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>120</y>
     <width>150</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>SCORE : </string>
   </property>
  </widget>
  <widget class="QLabel" name="score_label_2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>210</y>
     <width>150</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>STEPS : </string>
   </property>
  </widget>
  <widget class="QLabel" name="score_label_3">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>30</y>
     <width>150</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>BEST : </string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
